lib/crop.js

```diff
@@ -24,31 +32,36 @@ class Crop extends DarkroomStream {
 
     const originalBuffer = Buffer.concat(this.chunks, this.size)
     if (imageFileType.ext !== 'gif') {
-      context = gm(originalBuffer)
-        .options({ nativeAutoOrient: true })
-        .autoOrient()
-        .colorspace('RGB')
-        .gravity(this.gravity)
-        .noProfile()
-        .bitdepth(8)
-        .unsharp(0.25, 0.25, 8, 0.065)
-        .crop(this.crop.w, this.crop.h, this.crop.x1, this.crop.y1)
-
-      if (imageFileType.ext !== 'png') {
-        context = context.quality(this.quality)
+      const coords = this._getRoundedCoords()
+      context = sharp(originalBuffer)
+        .rotate()
+        .toColorspace('RGB')
+        .sharpen({ m1: 0.25, sigma: 0.25, y2: 8, x1: 0.065 })
+        .extract({
+          left: coords.x1 || 0,
+          top: coords.y1 || 0,
+          width: coords.width,
+          height: coords.height
+        })
+
+      if (imageFileType.ext === 'png') {
+        context = context.png({ quality: this.quality })
       }
 
       if (imageFileType.ext === 'jpg') {
         // This makes it progressive
-        context = context.interlace('Line')
+        context = context.jpeg({ progessive: true, quality: this.quality })
       }
-      context.stream(this.output.bind(this))
+
+      // eslint-disable-next-line no-console
+      this.sharp(context)
     } else if (imageFileType.ext === 'gif') {
       // this is the rounding graphicsmagick uses for geometry
```

Comment:
```suggestion
      // this is the rounding sharp uses for geometry
```

Reply:
To be honest, I don't really know that it's true for sharp - maybe I'll just change it to say we used to have this for GM, so have kept it for sharp